### PR TITLE
feat: 🎸 Add Middleware V2 version compatibility check

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Adds middleware v2 version compatibility check with SDK

### Breaking Changes

NA

### JIRA Link

DA-763

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Comment by prashantasdeveloper** [comment]
Note: this will get merged when we merge all alpha SQ changes to master. Once that is done, we will have the new minimum version and that will be our base moving forward

File: src/utils/constants.ts [hunk 1]
```diff
@@ -136,3 +136,8 @@ export const MAX_LEGS_LENGTH = 10;
  * Default CDD ID associated with an Identity on chain. Used for Identities onboarded without PUIS
  */
 export const DEFAULT_CDD_ID = '0x0000000000000000000000000000000000000000000000000000000000000000';
+
+/**
+ * Minimum version of Middleware V2 GraphQL Service (SubQuery) that is compatible with this version of the SDK
+ */
+export const MINIMUM_SQ_VERSION = '9.7.0-alpha.2';
```
Comment (polymath-eric) [inline src/utils/constants.ts:143]: Just want to make sure, does the semver library work with "alpha" tag, as in if we had SQ version `9.8.0`, would things still be OK.

**Review by polymath-eric — APPROVED** [review]
lgtm

File: src/utils/constants.ts (hunk 1 above)
Comment (prashantasdeveloper) [reply src/utils/constants.ts]: Yes, this works correctly

**Pull request merged**